contributing/standards.md

```diff
@@ -0,0 +1,18 @@
+## Brownbook Standards and Review Checklist
+
+Try to follow this list of standards when writing content for brownbook. Reviewers can and should refer to this list while reviewing brownbook content.
+
+### General
+* Content should be free of spelling, grammar, punctuation, or formatting errors.
+
+### "Meta"
+* Use "brownbook" as a proper noun, not "the brownbook"
+* Brownbook is not capitalized unless it is in a title or at the beginning of a sentence.
```

Comment:
This helps build the brand!

Reply:
Lol i just realized this is like facebook but i didn't intend it from the start 😂